tsfile/src/main/java/org/apache/iotdb/tsfile/write/record/Tablet.java

```diff
@@ -95,12 +103,48 @@ public Tablet(String deviceId, List<MeasurementSchema> schemas, int maxRowNumber
     this.deviceId = deviceId;
     this.schemas = schemas;
     this.maxRowNumber = maxRowNumber;
+    mschemawithindex=new HashMap<>();
+
+    for(int i=0;i<schemas.size();i++){
```

Comment:
code should be 
`for (int i = 0; i < schemas.size(); i++) {}`.  It's not wasteful to add blank in the code.

Reply:
copy that,thank you